packages/host/src/brain.rs

```diff
@@ -185,7 +185,7 @@ impl Brain {
 
                 display: Display::new(peripherals_tx.clone(), display_rx),
             }),
-            link_addr: link_addr_2.clone(),
+            link_addr: link_addr_2,
```

Comment:
This can be inlined as `link_addr.clone()` rather than making a variable just for the clone.